src/pre_defined_classes/mathematical_program/medium/q3/TrigIdentityTest.java

```diff
@@ -0,0 +1,40 @@
+package pre_defined_classes.mathematical_program.medium.q3;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TrigIdentityTest extends BaseTest {
+    final double cos2 = 0.2500000000000001;
+    final double sin2 = 0.7499999999999999;
+    final double sum = 1.0;
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The value of sin\\^2 is\\: "),
+                new DoubleLiteral(0, 1.0, "sin2"),
+                new NewLine(),
+                new StringLiteral("The value of cos\\^2 is\\: "),
+                new DoubleLiteral(0, 1.0, "cos2"),
+                new NewLine(),
+                new StringLiteral("Their sum is\\: "),
+                new DoubleLiteral(0, 2.0, "sum")
+        };
+    }
+
+    public void runMain() {
+        TrigIdentity.main(new String[0]);
+    }
+
+    @Test
```

Comment:
Based on the fact that there are no distractors and that the way the program is set up, it won't compile if the math is out of order, they can't really get the math wrong. It might be more useful to say something like "Your first output value should be sin^2"

Reply:
I don't know, Opey said that we should still check the math

Reply:
But I could add a separate test, or we would provide the format output

Reply:
And it actually won't pass the clause test if it is in the wrong order, so we just need to provide the output format

Reply:
I figured it might give them a better idea of what order to put them in, but it might not be necessary.